robot_control:
  type: position_controllers/JointTrajectoryController
  joints:
    - Joint1
    - Joint2
    - Joint3
    - Joint4
    - Joint5
  gains:
    Joint1:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    Joint2:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    Joint3:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    Joint4:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    Joint5:
      p: 100
      d: 1
      i: 1
      i_clamp: 1